src/node/utils/Minify.js

```diff
@@ -406,10 +406,7 @@ function getFile(filename, callback) {
 
 function compressJS(content)
 {
-  var decoder = new StringDecoder('utf8');
-  var code = decoder.write(content); // convert from buffer to string
-  var codeMinified = uglifyJS.minify(code, {fromString: true}).code;
-  return codeMinified;
+  return UglifyJS.minify(content);
```

Comment:
This is wrong.

On top of the file uglifyjs is imported with a lowercase letter.
```js
var uglifyJS = require("uglify-js");
```

How did it work?

I will force push on develop.